### DISCUSSION TITLE
Cannot find this box in Flows

### DISCUSSION THREAD

**Discussion by pihentagy** [postmanlabs/postman-flows#398] [Q&A] [answered]
What is the name of this block (found in 6 - Pagination)? I am not able to create such thing in flows.

![image](https://user-images.githubusercontent.com/29916/191499506-dad99218-5d46-4dd1-979c-03060a60341b.png)

**Comment by shrey-bhagat** [comment, marked as answer]
hi @pihentagy this is `select` block
